prac_06/programming_language.py

```diff
@@ -0,0 +1,21 @@
+"""Class defining a Programming Language."""
+
+
+class ProgrammingLanguage:
+    """Represents a Programming Language object."""
+
+    def __init__(self, name, typing, reflection, year_created):
+        """Initialise a Programming Language instance."""
+        self.name = name
+        self.typing = typing
+        self.reflection = bool(reflection)
+        self.year_created = int(year_created)
+
+    def __str__(self):
+        """String form of the Programming Language instance."""
```

Comment:
You could write your function docstring as """**Return** string form of the Programming Language instance.""" since the function's task is to return something.

Reply:
Missed this one when I was editing my docstrings. Thanks.